build.sbt

```diff
@@ -152,7 +152,8 @@ lazy val releaseSettings = Seq(
     // current maintainers
     Developer(id="anne-decusatis", name="Anne DeCusatis", email="anned@spotify.com", url=url("https://twitter.com/precisememory")),
     Developer(id="catherinejelder", name="Catherine Elder", email="siege@spotify.com", url=url("https://twitter.com/siegeelder")),
-    Developer(id="idreeskhan", name="Idrees Khan", email="me@idreeskhan.com", url=url("https://twitter.com/idreesxkhan")),
+    Developer(id="idreeskhan", name="Idrees Khan", email="me@idrees.xyz", url=url("https://twitter.com/idreesxkhan")),
+//    Developer(id="jackdingilian", name="Jack Dingilian", email="jackd@spotify.com", url=url("")),
```

Comment:
@jackdingilian have a URL to put here? it errors on malformed if I leave it empty